pkgs/tools/security/das/default.nix

```diff
@@ -5,19 +5,20 @@
 
 python3.pkgs.buildPythonApplication rec {
   pname = "das";
-  version = "0.3.6";
+  version = "0.3.8";
   format = "pyproject";
 
   src = fetchFromGitHub {
     owner = "snovvcrash";
     repo = "DivideAndScan";
     rev = "refs/tags/v${version}";
-    hash = "sha256-UFuIy19OUiS8VmmfGm0F4hI4s4BU5b4ZVh40bFGiLfk=";
+    hash = "sha256-a9gnEBTvZshw42M/GrpCgjZh6FOzL45aZqGRyeHO0ec=";
   };
 
   postPatch = ''
     substituteInPlace pyproject.toml \
-      --replace 'networkx = "^2.8.4"' 'networkx = "*"'
+      --replace 'networkx = "^2.8.4"' 'networkx = "*"' \
+      --replace 'pandas = "^1.4.2"' 'pandas = "*"'
   '';
```

Comment:
I think this entire `postPatch` block could be replaced with the (more declarative) `pythonRelaxDepsHook`:
```nix
{
  # snip...
  nativeBuildInputs = [ pythonRelaxDepsHook ];
  pythonRelaxDeps = [ "networkx" "pandas" ];
  # snip...
}
```

Reply:
Thank you for the super fast review!

I tried it and it doesn't seem to work here. I've heard on Matrix that this approach doesn't work on poetry projects for some reason, but I don't actually know why.